src/sider/frisktilarbeid/FriskmeldingTilArbeidsformidling.tsx

```diff
@@ -1,16 +1,23 @@
 import React, { ReactElement } from "react";
 import { useVedtakQuery } from "@/data/frisktilarbeid/vedtakQuery";
 import { VedtakResponseDTO } from "@/data/frisktilarbeid/frisktilarbeidTypes";
+import { FerdigbehandletVedtak } from "@/sider/frisktilarbeid/FerdigbehandletVedtak";
 import { VedtakFattet } from "@/sider/frisktilarbeid/VedtakFattet";
-import { FattVedtak } from "@/sider/frisktilarbeid/FattVedtak";
+import { FattNyttVedtak } from "@/sider/frisktilarbeid/FattNyttVedtak";
 
 export const FriskmeldingTilArbeidsformidling = (): ReactElement => {
   const { data } = useVedtakQuery();
   const vedtak: VedtakResponseDTO | undefined = data[0];
+  const isFerdigbehandlet = !!vedtak?.ferdigbehandletAt;
+  const isExistingVedtak = !!vedtak;
 
-  if (!vedtak) {
-    return <FattVedtak />;
+  if (isExistingVedtak) {
```

Comment:
Sikkert en test som kan utvides til å sjekke at riktig komponent vises basert på logikken her

Reply:
Lurt! La til test for å sjekke at vi rendrer vedtak som veileder har ferdigbehandlet :)